docs/buildah.md

```diff
@@ -1,18 +1,17 @@
-## buildah "March 2017"
+## buildah "1" "March 2017" "buildah"
 
 ## NAME
-buildah - A command line tool to facilitate working with containers and their images. 
-
+buildah - A command line tool to facilitate working with containers and using them to build images.
 
 ## DESCRIPTION
 The buildah package provides a command line tool which can be used to:
 
     * Create a working container, either from scratch or using an image as a starting point.
-    * Mount the working container's root filesystem for manipulation.
-    * Unmount the working container's root filesystem.
-    * Use the updated contents of the container's root filesystem as a filesystem layer to create a new image.
-    * Delete a working container or it's image.
- 
+    * Mount a working container's root filesystem for manipulation.
```

Comment:
FWIW, I lifted most of this list from the main README.md.  You might want to touch that up too in this or another PR.